### DISCUSSION TITLE
USER AUTHENTICATION AND LOGIN DISCUSSION

### DISCUSSION THREAD

**Discussion by Luson045** [Luson045/medi-connect#40] [Ideas]
<h1>EVERYONE WORKING ON USER AUTH OR SOMETHING RELATED TO THIS PLEASE WRITE YOUR IDEAS HERE ELSE ILL HAVE TO CLOSE YOUR ISSUE</h1>

**Comment by 29deepanshutyagi** [comment]
kindly go through issue #39

**Comment by Luson045** [comment]
okay so here's the plan,
@29deepanshutyagi  will implement zod for user verification
OAUTH will be used without zod for direct signup
phone-no verification through OTP or something will be independent of zod
and forget password email will also be independent

**Comment by varma-101** [comment]
I will implement authentication & authorization. Store the password using hashing technique in passport.js. This should be done in parallel with @29deepanshutyagi as to authenticate and store password in proper form.

**Comment by ShaikHafiza** [comment]
Then i would work on setting password requirements(i.e, min 8 words, do not contains spaces etc). and display the corresponding alerts accordingly , to align with @varma-101 and @29deepanshutyagi .

**Comment by 29deepanshutyagi** [comment]
sure

**Comment by Luson045** [comment]
Great, as these issues revolve very close to each other's, we need to keep in mind to handle it carefully 👍🏻

**Reply by varma-101** [reply]
Glad to hear, we will work on these well.
